docs/production/upgrade-or-modify.md

```diff
@@ -331,6 +335,10 @@ That last command will finish by restarting your Zulip server; you
 should now be able to navigate to its URL and confirm everything is
 working correctly.
 
+At this point, we recommend [upgrading from Xenial to
+Bionic](#upgrading-from-ubuntu-16-04-xenial-to-18-04-bionic), so that
+you are running a supported operating system.
```

Comment:
I think this one should probably point to the "Upgrade Xenial to Bionic" instead.

And what do you think about making this numbered as step 6?  I think it might be helpful, as it will be a step required to have a secure Zulip server again.

Reply:
It does point to "Upgrade Xenial to Bionic", I believe -- both in anchor and in words.

I didn't make this step 6 because it'd also require reworking the previous paragraph.  In the case of this Trusty -> Xenial, it does make sense in terms of being required -- but it's in some ways odd to have "upgrade to Bionic" as a step that is part of the heading "Upgrading from Trusty to Xenial."

If we're going to go that route, the header should just be "Upgrading from Trusty."

Reply:
Hmm, not sure I like that header change; that block has the instructions for upgrading from Trusty and then a pointer to the "Upgrading from Xenial" docs.  But yeah #15877 is what was on my mind.

BTW we should remember to backport these commits to the `3.x` branch so they update http://zulip.readthedocs.io/en/stable/.